db/salivary_gland.yml

```
@@ -30,35 +30,80 @@
 
     - name: CDK12
       types:
-        - truncation
+        - amplification
 
-  optional-markers:
     - name: TP53
       types:
         - truncation
 
-    - name: NF1
+  optional-markers:
+    - name: CDKN2A
       types:
-        - p.K1444E
-        - p.R1276P
+        - loss
 
+- type: Salivary Gland
+  subtype: Salivary Duct Carcinoma
+  required-markers:
+    - name: HRAS
+      types:
+        - amplification
```

```suggestion
        - amplification
```
加藤先生のコメントでは、「HRASの変異も多いようです」となっていますが、ampでよいでしょうか？